index.html

```diff
@@ -80,366 +219,415 @@
-    if ('creation' in data)
-        text += 'Creation: ' + gasToText(data.creation[0]) + ' + ' + gasToText(data.creation[1]) + '\n';
-    text += 'External:\n';
-    for (var fun in data.external)
-        text += '  ' + fun + ': ' + gasToText(data.external[fun]) + '\n';
-    text += 'Internal:\n';
-    for (var fun in data.internal)
-        text += '  ' + fun + ': ' + gasToText(data.internal[fun]) + '\n';
-    return text;
-};
-var formatAssemblyText = function(asm, prefix, source) {
-    if (typeof(asm) == typeof('') || asm === null || asm === undefined)
-        return prefix + asm + '\n';
-    var text = prefix + '.code\n';
-    $.each(asm['.code'], function(i, item) {
-        var v = item.value === undefined ? '' : item.value;
-        var src = '';
-        if (item.begin !== undefined && item.end != undefined)
-            src = source.slice(item.begin, item.end).replace('\n', '\\n', 'g');
-        if (src.length > 30)
-            src = src.slice(0, 30) + '...';
-        if (item.name != 'tag')
-            text += '  ';
-        text += prefix + item.name + ' ' + v + '\t\t\t' + src +  '\n';
-    });
-    text += prefix + '.data\n';
-    if (asm['.data'])
-        $.each(asm['.data'], function(i, item) {
-            text += '  ' + prefix + '' + i + ':\n';
-            text += formatAssemblyText(item, prefix + '    ', source);
+
+    <div id="editor">
+        <div id="input"></div>
+    </div>
+
+    <div id="righthand-panel">
+        <div id="dragbar"></div>
+        <div id="header">
+            <img id="solIcon" src="solidity.svg">
+            <h1>Solidity realtime<br/>compiler and runtime</h1>
+            <div class="info">
+                <p>Version: <span id="version">(loading)</span><br/>
+                Execution environment does not connect to any node, everyhing is in-memory only.<br/>
+                <strong>Note:</strong> If Chrome/Chromium reports &quot;Uncaught JavaScript Exception&quot;,
+                enable the debug console (Ctrl+Shift+i) and reload.</p>
+            </div>
+            <div id="optimizeBox">
+                <input id="editorWrap" type="checkbox"><label for="editorWrap">Text Wrap</label>
+                <input id="optimize" type="checkbox"><label for="optimize">Enable Optimization</label>
+            </div>
+        </div>
+        <div id="output"></div>
+    </div>
+
+    
+
+    <script>
+
+
+
+        // ----------------- resizeable ui ---------------
+
+       var dragging = false;
+       $('#dragbar').mousedown(function(e){
+           e.preventDefault();
+           
+           dragging = true;
+           var main = $('#righthand-panel');
+           var ghostbar = $('<div id="ghostbar">', {
+                css: {
+                    top: main.offset().top,
+                    left: main.offset().left
+                }
+            }).prependTo('body');
+           
+            $(document).mousemove(function(e){
+              ghostbar.css("left",e.pageX+2);
```

Comment:
Sure can, does this feature help with your feedback one previous, re default sizes?